content/en/docs/userguide/locality_loadbalance.md

```diff
@@ -0,0 +1,244 @@
+---
+draft: false
+linktitle: Locality Load Balancing
+menu:
+  docs:
+    parent: user guide
+    weight: 20
+title: Locality Load Balancing
+toc: true
+type: docs
+
+---
+
+This document introduces how to use Locality Load Balancing with Istio in the Kmesh.
+
+> The current Kemsh Locality Load Balancing is at the L4 level and only support [Locality Failover](https://istio.io/latest/docs/tasks/traffic-management/locality-load-balancing/failover/).
+
+#### What is Locality Load Balancing?
+A locality defines the geographic location of a workload instance within mesh. Locality Load Balancing in service mesh helps improve the availability and performance of services by intelligently routing traffic based on the location of the service instances.
+
+We strongly recommend that you first read https://istio.io/latest/docs/tasks/traffic-management/locality-load-balancing/ to understand what locality load balancing is.
+
+#### Supported Modes and Configuration Methods for Kmesh:
+Currently, Istio's ambient mode only supports specifying a fixed locality load balancing policy by configuring specific fields. This includes two modes: PreferClose and Local.
+
+1. PreferClose: A failover mode that uses NETWORK, REGION, ZONE, and SUBZONE as the routingPreference.
+```
+&workloadapi.LoadBalancing{
+	// Prefer endpoints in close zones, but allow spilling over to further endpoints where required.
+	RoutingPreference: []workloadapi.LoadBalancing_Scope{
+		workloadapi.LoadBalancing_NETWORK,
+		workloadapi.LoadBalancing_REGION,
+		workloadapi.LoadBalancing_ZONE,
+		workloadapi.LoadBalancing_SUBZONE,
+	},
+	Mode: workloadapi.LoadBalancing_FAILOVER,
+}
+```
+  - With `spec.trafficDistribution`（k8s >= beta [1.31.0](https://kubernetes.io/docs/concepts/services-networking/service/), isito  >= [1.23.1](https://istio.io/latest/news/releases/1.23.x/announcing-1.23/)）
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+  namespace: default
+spec:
+  trafficDistribution: # spec.trafficDistribution
+    preferClose: true
+```
+  - With annotation [todo: need to verify the feasibility.]
+```
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    networking.istio.io/traffic-distribution: PreferClose
+```
+todo: 没有在istio的api.annotation找到traffic-distribution这个标签，但是istio的server.go显示了通过这个字段配置策略的可能性。
+
+2. Local: A strict mode that only matches the current NODE.
+```
+&workloadapi.LoadBalancing{
+  // Only allow endpoints on the same node.
+  RoutingPreference: []workloadapi.LoadBalancing_Scope{
+    workloadapi.LoadBalancing_NODE,
+  },
+  Mode: workloadapi.LoadBalancing_STRICT,
```

Comment:
This canbe deleted, only need to mention the visible results with strict setting